src/Raven.Studio/typescript/models/database/tasks/periodicBackup/s3Settings.ts

```diff
@@ -1,14 +1,19 @@
 ﻿import amazonSettings = require("models/database/tasks/periodicBackup/amazonSettings");
 import jsonUtil = require("common/jsonUtil");
+import genUtils = require("common/generalUtils");
 
 class s3Settings extends amazonSettings {
     bucketName = ko.observable<string>();
-
+    useCustomS3Host = ko.observable<boolean>(true);
```

Comment:
why true by default ?!

Reply:
oh you override this in constuctor - so please skip default value here

Reply:
removed

Reply:
ok